properties.json

```diff
@@ -0,0 +1,644 @@
+[
+  {
+    "nameStr": "Go",
+    "cost": null,
+    "rent": null,
+    "houseCost": null,
+    "mortgage": null,
+    "ownerID": null,
+    "houses": null,
+    "isMortgaged": null,
+    "isBuyable": null,
+    "functionID": 6
```

Comment:
I'm only guessing here but functionID probably refers to a numeric ID which executes the logic for when someone lands on this space?  

If that's so, I'd go ahead an replace this with a string indicating the logic that should be executed.

type: "GO", type: "Property', type: "utility", etc. etc.

The extremely marginal speed increase you're getting by dealing with integers instead of strings isn't worth the confusion and burden of knowledge necessary to work with this JSON object.  

Jacob Gardner's Pillars of Development in order
 - Write Self-Documenting Code
 - Write Clear Code
 - Write code that reduces burden of knowledge (see 1 and 2)
 - Write fast code (supercedes top 3 during optimization stage and replacement is very well documented)